app/concepts/position.rb

```diff
@@ -0,0 +1,18 @@
+class Position
+  attr_reader :stock, :total_quantity, :market_price, :cost
+
+  def initialize(stock:, quantity:, market_price:, cost:)
+    @stock = stock
+    @total_quantity = quantity
+    @market_price = market_price
+    @cost = cost
+  end
+
+  def total_value
+    (total_quantity || 0) * (market_price || 0)
+  end
+
+  def percentage_change
+    (((total_value.to_f / cost.to_f) * 100) - 100).round(0)
```

Comment:
If we're working with monetary values, we should probably use decimals instead of floats.